hw/ip/clkmgr/data/clkmgr.hjson.tpl

```diff
@@ -251,11 +251,20 @@
           name: "HI_SPEED_SEL",
           mubi: true,
           desc: '''
-            A value of kMultiBitBool4True selects nominal speed external clocks.
+            A value of kMultiBitBool4True selects nominal speed external clock.
             All other values selects low speed clocks.
 
             Note this field only has an effect when the !!EXTCLK_CTRL.SEL field is set to
             kMultiBitBool4True.
+
+            Nominal speed means the external clock is approximately the same frequency as
+            the internal oscillator source.  When this option is used, all clocks operate
+            at roughly the nominal frequency.
+
+            Low speed means the external clock is apporximately half the frequency as the
+            internal oscillator source. When this option is used, previously undivided clocks
+            now run at half frequency.  Previously divided clocks will halve the divider and
+            run at roughly the nomianl frequency.
```

Comment:
Should this be "quarter the nominal frequency"? (Also, spelling of "nominal")

Reply:
ah shoot... let me add more explanation.  The idea is that the dividers are stepped down, so even though the source frequency is halved, the divider output remains the same. it's....weird.

Reply:
let me know if the update is clearer.

Reply:
Ah! So "stepped down" here means that the dividers don't divide by as much? I think I understand now (and have just read the relevant part of the documentation). Maybe the best thing is just to include a note to look at the "Software Requested External Clocks" section?

Reply:
yep! i've added that as well.